config/rollup/index.js

```diff
@@ -50,7 +50,11 @@ module.exports = function createRollupConfig({entry, cssPath}) {
         exclude: 'node_modules/**',
         runtimeHelpers: true,
       }),
-      commonjs(),
+      commonjs({
+        namedExports: {
```

Comment:
I had trouble exporting `TestProvider` because of an issue with `createMount`. Here I'm following the docs ([rollup docs](https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module), [github rollup docs](https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports))